notebooks/DrizzlePac/align_sparse_fields/align_sparse_fields.ipynb

```diff
@@ -4,14 +4,52 @@
    "cell_type": "markdown",
```

Comment:
<p>Let's get the path with a more readable way and check if <code>mastDownload</code> exist before removing.</p><p> </p>

<pre class="ql-syntax" spellcheck="false">for flc in glob.glob("./mastDownload/HST/*/*flc.fits"):
&nbsp;&nbsp;flc_name = os.path.basename(flc)
&nbsp;&nbsp;os.rename(flc, flc_name)
if os.path.exists("mastDownload"):
&nbsp;&nbsp;shutil.rmtree("mastDownload")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>